lang/java/reef-wake/wake/src/main/java/org/apache/reef/wake/remote/transport/netty/ChunkedReadWriteHandler.java

```diff
@@ -147,9 +147,9 @@ public void write(final ChannelHandlerContext ctx, final Object msg, final Chann
    */
   private byte[] sizeAsByteArr(final int size) {
     final byte[] ret = new byte[INT_SIZE];
-    final ByteBuf intBuffer = Unpooled.wrappedBuffer(ret).order(Unpooled.LITTLE_ENDIAN);
+    final ByteBuf intBuffer = Unpooled.wrappedBuffer(ret);
```

Comment:
Any particular reason not to use `order`?

Reply:
The complier warned me that `ByteBuf.order` has deprecated. I followed the instruction at [Netty docs](https://netty.io/4.1/api/io/netty/buffer/ByteBuf.html#order-java.nio.ByteOrder-).